### ISSUE TITLE
[TSDoc> Search]: Search Suggestions list items are not accessible by the Up/Down arrow key.

### ISSUE THREAD

**Issue by vagpt** [microsoft/rushstack-websites#245]
**Test Environment:**
Version: Version 24H2 (OS Build 26231.5000)
Browser Version: 127.0.2638.2 (Official build) dev (64-bit)
URL: https://tsdoc.org/

**Pre-Requisite:**
1. Navigate through keyboard only.

**Repro Steps:**

1. Open the above URL in edge dev browser and login with v-.
2. Navigate to the 'Search' button and press enter key to activate it.
3. Type some characters, press tab key, and observe the issue.

**Actual Result:**
While navigating the suggestions list items, List items are not navigable through the Up and Down arrow keys. It is navigable through the Tab key only.

**Expected Result:**
List items should be navigable through the Up and Down arrows keys only. It should not navigable through the tab keys,

**User Impact:**
It will impact the keyboard user as they will try to navigate the list item with the up and down arrow keys only.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

https://github.com/microsoft/rushstack-websites/assets/95913748/5fa8ef07-fea3-47a0-a095-a8152eb6cacd

![List items are not navigable through the up and down arrow keys](https://github.com/microsoft/rushstack-websites/assets/95913748/3aab173f-871b-4d15-9fae-ec27846942e2)
